x-pack/metricbeat/module/aws/s3_request/_meta/fields.yml

```diff
@@ -0,0 +1,74 @@
+- name: s3_request
+  type: group
+  description: >
+    `s3_request` contains request metrics that were scraped from AWS CloudWatch which contains monitoring metrics sent by AWS S3.
+  release: beta
+  fields:
+    - name: bucket.name
+      type: keyword
+      description: >
+        Name of a S3 bucket.
+    - name: all_requests
```

Comment:
Maybe create a field group `request` or nothing at all and then put inside `get`, `put`... so you don't have a fields like `aws.s3_request.all_requests` and, instead, you have `aws.s3_request.requests.all` or even `aws.s3_request.all`, `aws.s3_request.get`

Reply:
I think that `aws.s3_request.total` instead of `aws.s3_request.all` is more widely used in Metricbeat

Reply:
Good point!! `requests.total` it is!!